.gitignore

```diff
@@ -53,3 +53,4 @@ config/database.yml
 vcr.log
 
 .byebug_history
+config/application.yml
```

Comment:
yeah, it will just be ignored - but if you want maps to work locally you'll need a new key in application.yml - and that's what we need to add docs about how to do